src/course7/homework/Challenge6.java

```diff
@@ -0,0 +1,24 @@
+package course7.homework;
+
+import java.util.Scanner;
+
+public class Challenge6 {
+
+    public static void main(String[] args) {
+
+        System.out.println("This program will continue to add numbers to a total until you enter -1 and after that it will show you" +
+                "the final result.");
+
+        Scanner scan = new Scanner(System.in);
+
+        double sum = 0;
+        double number = 0;
+        while (number != -1) {
+
+            System.out.println("Enter a new number.");
+            number = scan.nextDouble();
+            sum = sum + number;
+        }
+        System.out.println("Your final result is: " + sum);
```

Comment:
hmm.. so I am entering numbers 5 and 4, then -1. According to challenge requirement I would expect -1 to not count on the sum and to have the total as 9, but actually I get the result 8 (-1 counts): "... After they enter -1, print the sum of all numbers entered (excluding -1)."
Can you make it work as expected? :p

Reply:
Yup. It's done. I'll upload it shortly.